ush/python_utils/run_command.py

```diff
@@ -4,10 +4,10 @@
 
 
 def run_command(cmd):
-    """Run system command in a subprocess
+    """Runs system command in a subprocess
 
     Args:
-        cmd: command to execute
+        cmd (str): command to execute
```

Comment:
Should the c in command be capitalized?

```suggestion
        cmd (str): Command to execute
```